assignments/week-13/assign11.html

```diff
@@ -0,0 +1,9 @@
+1. const largestOfThree = (a, b, c) => (a > b && a > c) ? a : (b > a && b > c) ? b : c;
+2. const fetchData = (url) => fetch(url)
+  .then(response => response.json())
+  .then(data => console.log(data))
+  .catch(error => console.error('Error:', error));
+ 
+3. It calucates the sum of all integers from 1 to whatever is inputted for n 
+
+4. it errors because the sum is too high for what is inputted to be the maximum of what is allowed in the program. 
```

Comment:
It's not exactly that the sum is too high.  It is that the recursion goes too deep and the call stack runs out of memory.